lib/Doctrine/Common/Collections/ExpressionBuilder.php

```diff
@@ -40,19 +40,19 @@ class ExpressionBuilder
      *
      * @return CompositeExpression
      */
-    public function andX($x = null)
+    public function andX(...$x)
```

Comment:
sorry, I confused the Criteria system in doctrine/collection with the expression builder of the QueryBuilder. Forget my comment.